components/profile/components/index.js

```diff
@@ -0,0 +1,5 @@
+import ViewProfile from "./ViewProfile";
+import EditProfile from "./EditProfile";
+
+export { ViewProfile, EditProfile };
```

Comment:
i am surprised you are not getting an eslint error for not having this object sorted alphabetically

Reply:
I didn't actually. Although I did for an object I placed in constants.js